pages/videos.tsx

```diff
@@ -0,0 +1,55 @@
+import { Layout } from "../components/Layout";
+import { Navbar } from "../components/Navbar";
+import { useState } from "react";
+import { videos } from "../data/videos";
+
+const Videos = () => {
+  const [selectedVideo, setSelectedVideo] = useState<number>(0);
```

Comment:
I feel like index based `useState` statements are dangerous 🤔  But I don't have an exact reason why. 
I'd say it's better if you used the youtubeId for this...